src/lint/rules/tests/indentation/in9.rs

```diff
@@ -21,9 +21,9 @@ pub static IN9_INCORRECT_CASE_INDENT: &str = "
 method some_switch(int arg) {
     switch(arg) {
       case ZERO:
-#if (asdd == 0) {
-          some_call();
-}
+        #if (asdd == 0) {
+        some_call();
+        }
           if (a) {
```

Comment:
Also this snippet and its expected amount of errors should be turned back to their previous state